### PR TITLE
fix(cli): Fixed cli exit code for empty array

### LINKED ISSUE #15: CLI is exiting with status 1 when no releases are found
Hi,

having a problem with this cli command.

When there are releases that can be generated, everything is fine. but if none are returned from the module, the cli is exiting with exit code 1 in [here](https://github.com/conventional-changelog/conventional-github-releaser/blob/master/cli.js#L108). There need to be a check for an empty array that exits with code 0.

Gonna open a PR for that in a few.

### PR DESCRIPTION
**Pull request by apinnecke**
Fixes #15
Before this fix the cli exited with status code 1 when getting an empty array.
Now it prints a message and exits with code 0.

### CODE REVIEW

**Comment by apinnecke** [comment]
Okay, CI is failing now because the project isn't configured to access the env vars from forks. Please validate green tests manually or share the token!

**Comment by stevemao** [comment]
> Okay, CI is failing now because the project isn't configured to access the env vars from forks.

How do I config this?

File: cli.js [hunk 1]
```diff
@@ -96,6 +96,11 @@ conventionalGithubReleaser({
     process.exit(1);
   }
 
+  if (0 === data.length) {
+    console.log('No GitHub releases created because no git tags available to work with.');
```
Comment (stevemao) [inline cli.js]: This should be only printed if verbose

**Comment by stevemao** [comment]
Thanks. Looks great.

**Comment by apinnecke** [comment]
> How do I config this?

Oh, just read that this isn't possible with Travis out of the box. Circle has this switch to give all forks access to the env vars. Guessing the best would be to add an encrypted env var to travis.yml as described [here](https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables).

File: cli.js (hunk 1 above)
Comment (apinnecke) [inline cli.js]: Yep, thanks for the hint, added the verbosity check in 283b3af.

**Comment by stevemao** [comment]
> Guessing the best would be to add an encrypted env var to travis.yml

I'm already doing it. Anything I'm missing?

**Comment by hbetts** [comment]
The linked Travis documentation specifically says - `Encrypted variables are not available to untrusted builds such as pull requests coming from another repository.`

Therefore there just isn't a way to support a successful build in a pull request.

**Comment by hbetts** [comment]
Can we do something like what I suggested [here](https://github.com/conventional-changelog/conventional-github-releaser/pull/12#issuecomment-255506433) to allow us to move forward with accepting this change?

**Comment by hbetts** [comment]
Once we've accepted #33 to get our CI tests working again I'll pull @apinnecke's branch into our repo and submit a pull request so that we can get a valid, and hopefully working, CI test.

**Comment by hbetts** [comment]
@apinnecke I would like to move forward with merging this in now, but here is a file conflict in `test/cli.js`.

Can you please resolve the file conflict and push an update.

Then I can push your changes as a branch on this repo, which will allow the end-to-end tests to run correctly.

**Comment by apinnecke** [comment]
@hbetts I've rebased my lasted master onto the current repo's master, should be fixed now. Could you please try again? :)

**Comment by hbetts** [comment]
@apinnecke I've push your code into a branch on the project so that we could get the end-to-end tests working. Please see #39.

However, your specific test is currently failing:

```
should work with no releases
Uncaught AssertionError: expected 1 to equal 0
```

Can you please take a look?

**Pull request merged**